networkit/test/test_graph.py

```diff
@@ -62,5 +63,46 @@ def testSubgraphFromNodes(self):
 		self.assertEqual(res.numberOfNodes(), 4)
 		self.assertEqual(res.numberOfEdges(), 4)
 
+	def test_graph_transpose(self):
```

Comment:
For reproducibility, I think that it would better to fix a random seed for both the `random` module and NetworKit (with `networkit.setSeed`).

Reply:
Yes.